code/src/App.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './Album'
+import './app.css'
+
 
 console.log(data)
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div >
+      <h1>New Albums and Releases</h1>
+      <div className="appcontainer">
+        {data.albums.items.map((album) => (
+          <Album
+            key={album.id}
+            image={album.images[1].url}
+            uri={album.external_urls.spotify}
+            artistname={album.artists[0].name}
+            artisturi={album.artists[0].external_urls.spotify}
+            albumname={album.name} />
```

Comment:
stick to camelCase for the prop names

Reply:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of `artistname` and `artisturi`go for `artists={alnum.artists}`